Simplenote.xcodeproj/project.pbxproj

```diff
@@ -4628,8 +4624,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/Automattic/XCUITestHelpers";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 0.1.0;
+				branch = "add-is-dark-mode";
+				kind = branch;
```

Comment:
This needs to change back to using the version before merging. Just waiting for https://github.com/Automattic/XCUITestHelpers/pull/1 to ship first.